RecoverVm/RecoverVM.ps1

```diff
@@ -0,0 +1,32 @@
+﻿Param(
+    [Parameter(Mandatory=$true)][string]$ServiceName ,
+    [Parameter(Mandatory=$true)][string]$VMName 
+)
+
+$Sub = Get-AzureSubscription -Current
+if ( ! $Sub ) 
+{
+    Write-Output "no current subscription set - please run add-azureaccount + select-azuresubscription -current first for the subscription containing your target vm!"
```

Comment:
Why don't we simply call these commands for the user instead of asking them to do it out of band?